pkg/utils/logging/statelog.go

```diff
@@ -11,6 +11,7 @@ import (
 )
 
 type msgMap [14][8]int
+type deathMap [8]int
 type treatyResponsesCount [2][8]int // 0 for reject, 1 for accept
```

Comment:
hate this lmao but fine given deadline i suppose

are we fixing the 14 to 12?

Reply:
uhh 14 to 12 is Daryl's code but it works so we didn't want to mess with it :))

Reply:
uhhh 14 to 12 works fine for now @daryllimyt and I didn't want to break something with it
are message types unused still showing in graphs?

Reply:
Yes, that PR hasn't been merged

Reply:
ohh, will look at it